docs/sources/dirtyduck/docs/index.md

```diff
@@ -0,0 +1,5076 @@
+```
+
+| entity<sub>id</sub> | license<sub>num</sub> | facility<sub>aka</sub> | facility<sub>type</sub> | activity<sub>period</sub> |
+|------------------- |--------------------- |---------------------- |----------------------- |------------------------- |
+| 229                 | 1596210               | food 4 less            | grocery store           | [2010-01-08,)             |
+| 355                 | 1874347               | mcdonalds              | restaurant              | [2010-01-12,2017-11-09)   |
+| 840                 | 1142451               | jewel foodstore # 3345 | grocery store           | [2010-01-26,)             |
+
+The first thing `triage` does when executes the experiment, is split the time that the data covers in blocks considering the time horizon for the *label* ( *Which facilities will fail an inspection in the following month?* in this scenario of **inspection prioritization<sup><a id="fnr.30" class="footref" href="#fn.30">30</a></sup>**) . This time horizon is calculated from a set of specific dates (`as_of_date` in triage parlance) that divide the blocks in past (for training the model) and future (for testing the model). The set of `as_of_dates` is (*mainly*) calculated from the *label timespan* and the *update frequency*<sup><a id="fnr.31" class="footref" href="#fn.31">31</a></sup>. The *as of date* is not the *event date*. The *event date* occurred *when* the facility was inspected. The *as of date* is when the planning of the future facilities to be inspected happens.
+
+`triage` will create those *labels* using information about the *outcome* of the event<sup><a id="fnr.32" class="footref" href="#fn.32">32</a></sup>, taking into account the temporal structure of the data. In our example: if a facility is inspected is an event, and whether it fails the inspection (outcome *true*) or not (outcome *false*).
+
+For a given entity on a given *as of date*, `triage` asks whether there's an outcome in the future time horizon. If so, `triage` will generate a *label* for that specific entity on that *as of date*.
+
+For this example, the *label* will be if given an *as of date* (e.g. January first, 2014), the facility will have a failed inspection in the following year.
+
+The following example hopefully will clarify the difference between *outcome* and *label*. We will focus on events (inspections) that happened in the year of 2014.
+
+```sql
+select
+    date,
+    entity_id,
+    (result = 'fail') as outcome
+from
+    semantic.events
+where
+    '[2014-01-01, 2015-01-01]'::daterange @> date
+    and
+    entity_id in (229,355,840)
+order by
+    date asc;
+```
+
+| date       | entity<sub>id</sub> | outcome |
+|---------- |------------------- |------- |
+| 2014-01-14 | 840                 | f       |
+| 2014-02-04 | 229                 | f       |
+| 2014-02-24 | 840                 | t       |
+| 2014-03-05 | 840                 | f       |
+| 2014-04-10 | 355                 | t       |
+| 2014-04-
```

Comment:
Also a broken link (that should be to an anchor on this page). Also present in the org mode version.